### ISSUE TITLE
Setting for searching in waste bin / backup group not working

### ISSUE THREAD

**Issue by Cos-mo** [Kunzisoft/KeePassDX#613]
Samsung Galaxy A71 with Android 10
KPDX version 2.7, installed from Google play store, German language
Database fresh created with KPDX, database version KPDX3.1 (KeePass 2)
Size 6 kb, stored locally (BTW: It would be nice, if KPDX would show the size in the app, e.g. in the settings)

I have in the database an entry in the waste bin, where the entry name and the username are unique in the database.

Wether or not I have the setting (see the title) activated or not, no search finds those waste bin entry.

Besides that, I do not see, what the "backup group" does mean in KPDX. In KP for Windows or a Linux fork backup groups get created and used for backing up changed database entries (the old entry version). In KPDX the history of the entry changes get shown inside of the entries themselves and old versions can get restored from there. (A very good solution, btw.) So a backup group seems to be superfluous and I would not find a way to use it and I have in KPDX never seen such a group.

**Comment by J-Jamet** [comment]
The behavior is updated the next time you open the database. I will try to improve this for real-time search updates.

> Besides that, I do not see, what the "backup group" does mean in KPDX.

It is a folder which corresponds to the recycle bin for KeePass 1 databases. Here you are using a KeePass 2 database, so it is not present. KeePass used this solution because it allows you to migrate from a V1 database to a V2 database. (V1 does not contain a history for each entry)

**Comment by Cos-mo** [comment]
> The behavior is updated the next time you open the database.

Sorry, I cannot follow. I cannot tell, how often the database has been locked and reopened since I created the said waste bin entry. The username, which I described above, is `telefon`. But every search for this word comes with a zero result. Of course I have turned the said option to off. Just to be sure I switched the setting to on again, of course as expected with the same result.

The explanation about backup group makes sense. thank you for that.
